interfaces/IBF-dashboard/src/app/services/event.service.ts

```diff
@@ -134,18 +127,18 @@ export class EventService {
 
   private onTriggerPerLeadTime = (timesteps) => {
     let firstKey = null;
-    Object.keys(timesteps).forEach((key) => {
-      if (timesteps[key] === '1') {
-        firstKey = !firstKey ? key : firstKey;
-      }
-    });
+    Object.keys(timesteps)
+      .sort((a, b) => (a > b ? 1 : -1))
```

Comment:
@SwatiTh I added this 'sort' line to make sure that the timesteps are in the right order. Because I think this is again a case where the random can change randomly somehow, because in my local it (correctly) said '3 days', while on ibf-test it (incorrectly) said '7 days'. With this change I'm making sure that it is always the lowest one: '3 days'.